Options.py

```diff
@@ -843,11 +844,11 @@ class OptionList(Option[typing.List[typing.Any]], VerifyKeys):
     # If only unique entries are needed and input order of elements does not matter, OptionSet should be used instead.
     # Not a docstring so it doesn't get grabbed by the options system.
 
-    default: typing.List[typing.Any] = []
+    default: typing.Union[typing.List[typing.Any], typing.Tuple[typing.Any, ...]] = ()
```

Comment:
Does line 842 need to have that same change?

Reply:
I see that lines 872 and 873 don't have the same type either, so I'm guessing this is fine based on precedents

Reply:
That's my guess as well, but I can change it if someone has an objection and explains why.

Reply:
There's another PR that tried to deal with the type of `default`, and it went stale because of the issues we ran into. This is fine.